ros/resources/v0/hosts.py

```diff
@@ -90,6 +91,8 @@ def get(self):
                 host['id'] = i.__dict__['id']
```

Comment:
From documentation, `object.__dict__` is A dictionary or other mapping object used to store an object’s (writable) attributes.
That's  why you are not getting new property inside `__dict__`.
Could you please change the way you are accessing the attributes to fix this?